scapy/contrib/bgp.py

```diff
@@ -117,25 +119,50 @@ def i2m(self, pkt, i):
         """"Internal" (IP as bytes, mask as int) to "machine"
         representation."""
         mask, ip = i
-        ip = socket.inet_aton(ip)
+        ip = socket.inet_aton(str(ip))
         return struct.pack(">B", mask) + ip[:self.mask2iplen(mask)]
 
     def addfield(self, pkt, s, val):
         return s + self.i2m(pkt, val)
 
     def getfield(self, pkt, s):
+        '''pkt: BGPNLRI_IPv4
+            s: bytes byte 0: network prefix, byte 1+: network
+            e.g.
+            s = b'\x18\x03\x03\x03' 24/3.3.3
+        '''
```

Comment:
use [sphinx markup](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-signatures)
```suggestion
        """
        :param pkt: BGPNLRI_IPv4
        :param s: bytes byte 0: network prefix, byte 1+: network
        
        example:
            s = b'\x18\x03\x03\x03' 24/3.3.3
        """"
```